es-model-archive/kf-es-model-v0.5.0_1.2.0/file_centric.mapping.json

```diff
@@ -0,0 +1,528 @@
+        }, 
+        "kf_id": {
+          "type": "keyword"
+        }, 
+        "latest_did": {
+          "type": "keyword"
+        }, 
+        "modified_at": {
+          "format": "yyyy-MM-dd HH:mm:ss.SSSSSS||yyyy-MM-dd HH:mm:ss||yyyy-MM-dd'T'HH:mm:ss.SSSSSS+HH:mm", 
+          "type": "date"
+        }, 
+        "participants": {
+          "properties": {
+            "affected_status": {
+              "type": "boolean"
+            }, 
+            "alias_group": {
+              "type": "keyword"
+            }, 
+            "available_data_types": {
+              "type": "keyword"
+            }, 
+            "biospecimens": {
+              "properties": {
+                "age_at_event_days": {
+                  "type": "integer"
+                }, 
+                "analyte_type": {
+                  "type": "keyword"
+                }, 
+                "composition": {
+                  "type": "keyword"
+                }, 
+                "concentration_mg_per_ml": {
+                  "type": "float"
+                }, 
+                "dbgap_consent_code": {
+                  "type": "keyword"
+                }, 
+                "external_aliquot_id": {
+                  "type": "keyword"
+                }, 
+                "external_sample_id": {
+                  "type": "keyword"
+                }, 
+                "kf_id": {
+                  "type": "keyword"
+                }, 
+                "ncit_id_anatomical_site": {
+                  "type": "keyword"
+                }, 
+                "ncit_id_tissue_type": {
+                  "type": "keyword"
+                }, 
+                "sequencing_center_id": {
+                  "type": "keyword"
+                }, 
+                "shipment_date": {
+                  "type": "keyword"
+                }, 
+                "shipment_origin": {
+                  "type": "keyword"
+                }, 
+                "source_text_anatomical_site": {
+                  "type": "keyword"
+                }, 
+                "source_text_tissue_type": {
+                  "type": "keyword"
+                }, 
+                "source_text_tumor_descriptor": {
+                  "type": "keyword"
+                }, 
+                "spatial_descriptor": {
+                  "type": "keyword"
+                }, 
+                "uberon_id_anatomical_site": {
+                  "type": "keyword"
+                }, 
+                "volume_ml": {
+                  "type": "float"
+                }
+              }, 
+              "type": "nested"
+            }, 
+            "consent_type": {
+              "type": "keyword"
+            }, 
+            "diagnoses": {
+              "properties": {
+                "age_at_event_days": {
+                  "type": "integer"
+                }, 
+                "diagnosis": {
+                  "type": "keyword"
+                }, 
+                "diagnosis_category": {
```

Comment:
`diagnosis_category` is not defined in `es_mapper.yaml`, that means it should be in `dataservice` API.

The question is: `diagnosis_category` appears in both under `diagnoses` and `participant`, is this the case in `dataservice` api?

Reply:
@junjun-zhang yes it appears in both places - each diagnosis has a `diagnosis_category` of cancer or structural birth defect.  The `diagnosis_category` under participant is an aggregated rollup of the individual diagnosis `diagnosis_category` into a single classification of birth defect, cancer, or birth defect and cancer